lib/cartage/rack.rb

```diff
@@ -32,7 +32,8 @@ def require_metadata(value = (arg = false; nil)) # rubocop:disable Style/Semicol
       private
 
       def default_require_metadata
-        (ENV['RAILS_ENV'] || ENV['RACK_ENV']).to_s !~ /\A(?:development|test)\z/i
+        environment = (ENV['RAILS_ENV'] || ENV['RACK_ENV'] || 'development').to_s
```

Comment:
I don’t think that we need the `#to_s` here; I’m pretty sure that ENV['foo'] can only return strings.